CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ### Added
 - `DUO137`: lint for insecure itsdangerous kwarg usage ([#36](https://github.com/duo-labs/dlint/issues/36))
 
+### Fixed
+- False positive for `DUO137`. ([#37](https://github.com/duo-labs/dlint/issues/39))
```

Comment:
Looks like this should be `[#39]` instead of `[#37]`

Reply:
You are right! Freudian type since the check code is 137

Reply:
https://github.com/duo-labs/dlint/pull/40/commits/fd7506cf95c103e7e75d0fc98413918d81e0f247